src/main/webapp/icons/LICENSE.txt

```diff
@@ -99,3 +99,11 @@ Original file: https://github.com/PyCQA/pylint/blob/master/doc/logo.svg
 
 The 24x24 and 48x48 icons were created from this file. No modifications were made, except
 the scale that was changed.
+
+------------------------------------------------------------------------------
+
+License for RootFramework Icons
+
+https://github.com/robotframework/visual-identity
```

Comment:
I think it should be *Creative Commons Attribution-ShareAlike 4.0 International License*

Reply:
Yes, it seems to be the case

Reply:
My preference is to point to the github repository in the event the robotframework project decides to change the license.

Reply:
Well actually that is why I want to have the actual license here. Otherwise we may have a conflict if they change the web site later on. This seems pedantic but image licensing is a big deal in the internet. 

As a compromise: add both lines, URL and license?

Reply:
Makes sense.  I updated the license.txt to indicate this.